python/adxl345_helper.py

```
@@ -0,0 +1,16 @@
+import time
+import board
+import busio
+import adafruit_adxl34x
+
+class ADXL345Sensor:
+    def __init__(self):
+        self.i2c = busio.I2C(board.SCL, board.SDA)
+        self.accelerometer = adafruit_adxl34x.ADXL345(self.i2c)
+        self.accelerometer.enable_motion_detection(threshold=18)
+
+    def get_acceleration(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_acceleration`

(ANN201)